projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -16,6 +16,13 @@
     "cy:run:mobile:ci": "cypress run --config-file cypress.ci.1905.json --spec \"cypress/integration/mobile/**/*\"",
     "cy:cds:run:ci:2005": "cypress run --config-file cypress.ci.2005.json --record --key $CYPRESS_KEY --tag \"2005,b2c,all-cds\" --spec \"cypress/integration/!(vendor|b2b)/**/*.e2e-spec.ts,cypress/integration/vendor/cds/**/*.e2e-spec.ts\"",
     "cy:run:regression:ci": "cypress run --config-file cypress.ci.1905.json --spec \"cypress/integration/regression/**/*\"",
+    "cy:run:smoke": "cypress run --spec \"cypress/integration/smoke/**/*\"",
+    "cy:run:smoke:ci:1905": "cypress run --config-file cypress.ci.1905.json --record --key $CYPRESS_KEY --tag \"1905,b2c,smoke\" --spec \"cypress/integration/smoke/**/*\"",
+    "cy:run:smoke:ci:2005": "cypress run --config-file cypress.ci.2005.json --record --key $CYPRESS_KEY --tag \"2005,b2c,smoke\" --spec \"cypress/integration/smoke/**/*\"",
+    "cy:run:smoke:ci:2005:b2b": "cypress run --config-file cypress.ci.b2b.json --record --key $CYPRESS_KEY --tag \"2005,b2b,smoke\" --spec \"cypress/integration/b2b/smoke/**/*\"",
+    "cy:run:smoke:ci:ccv2": "cypress run --config-file cypress.ci.ccv2.json --record --key $CYPRESS_KEY --tag \"ccv2,b2c,smoke\" --spec \"cypress/integration/smoke/**/*\"",
```

Comment:
We've dropped distinction to smoke and regression tests in `develop`. Now all e2e run for each PR in CI. PLease drop those obsolete commands.

Reply:
Removed